TimePicker/tests/TimePicker-specs.js

```diff
@@ -163,4 +195,44 @@ describe('TimePicker', () => {
 
 		expect(header).toBeNull();
 	});
+
+	test('"timeToLocaleString" method should return "null" for an "undefined" time', () => {
```

Comment:
please use escaped     \' insted of "

Reply:
I went over all the files I had to increased coverage and replace all of the " i found in test descriptions with /'.